src/components/courseSettings/ManageStaffPanel.tsx

```diff
@@ -58,26 +60,26 @@ const ManageStaffPanel = ({
     <Card className="mb-3">
       <CardHeader>
         <CardTitle tag="h5" className="mb-0">
-          Manage {course && course.name} Staff
+          Manage Staff <FontAwesomeIcon icon={faUsers} className="mr-2" />
```

Comment:
I like the icons! I think these should be all or nothing across the various settings pages. Also, consider placing them before the text instead of after.

Reply:
Sure, I can see if I can add some more icons to the Queue Settings page, and I'll move the icons before the text.